readthedocs/builds/constants_docker.py

```diff
@@ -32,38 +32,42 @@
             "2.7": "2.7.18",
             "3.6": "3.6.15",
             "3.7": "3.7.17",
-            "3.8": "3.8.18",
-            "3.9": "3.9.18",
-            "3.10": "3.10.13",
-            "3.11": "3.11.6",
-            "3.12": "3.12.0",
+            "3.8": "3.8.19",
+            "3.9": "3.9.19",
+            "3.10": "3.10.14",
+            "3.11": "3.11.9",
+            "3.12": "3.12.3",
             "miniconda3-4.7": "miniconda3-4.7.12",
+            "miniconda3-3.12": "miniconda3-3.12-24.1.2-0",
```

Comment:
This should include the 24.x in the name, 3.12 references the python version, not the conda version.